planner/core/testdata/plan_suite_out.json

```diff
@@ -1150,7 +1150,7 @@
       {
         "SQL": "select /*+ HASH_JOIN(@sel_1 t1), HASH_JOIN(@sel_2 t1) */ t1.b, t2.a, t2.aa from t t1, (select t1.a as a, t2.a as aa from t t1, t t2) t2 where t1.a = t2.aa;",
         "Plan": "RightHashJoin{TableReader(Table(t))->LeftHashJoin{IndexReader(Index(t.f)[[NULL,+inf]])->IndexReader(Index(t.f)[[NULL,+inf]])}}(Column#1,Column#25)->Projection",
-        "Hints": "USE_INDEX(@`sel_1` `t1` ), USE_INDEX(@`sel_2` `t1` `f`), USE_INDEX(@`sel_2` `t2` `f`), HASH_JOIN(@`sel_2` `t1`, `t2`), HASH_JOIN(@`sel_1` `t1`)"
+        "Hints": "USE_INDEX(@`sel_1` `t1` ), USE_INDEX(@`sel_2` `t1` `f`), USE_INDEX(@`sel_2` `t2` `f`), HASH_JOIN(@`sel_2` `t1`, `t2`), HASH_JOIN(@`sel_1` `t1`), HASH_JOIN(@`sel_2` `t2`)"
```

Comment:
HASH_JOIN(@`sel_2` `t2`) is redundant with HASH_JOIN(@`sel_2` `t1`, `t2`)?

Reply:
fixed